server/mergin/sync/utils.py

```diff
@@ -546,3 +547,21 @@ def is_supported_type(filepath) -> bool:
 def get_mimetype(filepath: str) -> str:
     """Identifies file types by checking their headers"""
     return magic.from_file(filepath, mime=True)
+
+
+def get_x_accel_uri(*url_parts):
+    """
+    Get the accell uri for the given url parts
```

Comment:
Would be good to write some example here as it seems bit confusing. 

How about this way:

- pass absolute path to file on disk (e.g. /data/projectA/file.txt)
- verify path starts with LOCAL_PROJECTS and then replace with '/download' and do it posix way